<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_widget</class>
 <widget class="QWidget" name="admin_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Log_Out">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登出</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Book">
    <attribute name="title">
     <string>书籍信息</string>
    </attribute>
    <widget class="QPushButton" name="Add_Book">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>萝莉体 第二版</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加书籍</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-40</y>
       <width>1161</width>
       <height>651</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/bachgroundconfirm.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>1141</width>
        <height>461</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>书籍名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书籍作者</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ISBN代码</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出版社</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>总本数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>已借出本数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>剩余本数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>价格</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>总借阅次数</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="BookName">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Writer">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ISBNCode">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Publisher">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="BookOrderType">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>90</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>以总借阅次数从高到低排序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>以总借阅次数从低到高排序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>以剩余本数从高到低排序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>以剩余本数从低到高排序</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="Search">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>萝莉体 第二版</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <zorder>frame</zorder>
    <zorder>Add_Book</zorder>
    <zorder>Search</zorder>
   </widget>
   <widget class="QWidget" name="User">
    <attribute name="title">
     <string>用户信息</string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-40</y>
       <width>1161</width>
       <height>651</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/bachgroundconfirm.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableWidget" name="User_Info_table">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>1121</width>
        <height>461</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>用户ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户真实姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户等级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户已经借阅书目</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户最大借阅数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>剩余金钱</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户邮箱</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QPushButton" name="Search_UserInfo">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>萝莉体 第二版</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RealName">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="UserId">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Email">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="UserOrderType">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>以用户等级从高到低排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>以用户等级从低到高排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>以剩余金钱从高到低排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>以剩余金钱从低到高排序</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>借阅/归还信息</string>
    </attribute>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-40</y>
       <width>1161</width>
       <height>651</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/bachgroundconfirm.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableWidget" name="Borrow_Return_table">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>1121</width>
        <height>461</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ISBN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书籍名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>起始日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>截止日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>是否归还</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QPushButton" name="PaintGraph">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>萝莉体 第二版</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>绘制借阅图像</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ISBN_Record">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Search_RecordInfo">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>萝莉体 第二版</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UserId_Record">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="BookName_Record">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="RecordOrderType">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>以时间为主次序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>以ISBN为主次序</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>Log_Out</zorder>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>Add_Book</tabstop>
  <tabstop>Log_Out</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>User_Info_table</tabstop>
  <tabstop>Borrow_Return_table</tabstop>
  <tabstop>PaintGraph</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
